src/GitHub.VisualStudio/Views/GitHubPane/TextViewCommandDispatcher.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace GitHub.VisualStudio.Views.GitHubPane
+{
+    /// <summary>
+    /// Used to filter commands send to <see cref="IVsTextView"/> dispatch them to a <see cref="Exec"/> event. 
```

Comment:
This sentence doesn't make sense grammatically. Also might be useful to say _why_ this class is needed in the `<remarks>`? That is, why do commands need to be filtered (and?) dispatched to a(n) `Exec` event?